Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/client/view/launch/InputFlagLayout.java

```diff
@@ -62,8 +62,8 @@ Sorina Pop (sorina.pop@creatis.insa-lyon.fr)
     private CheckboxItem cbFlagInputItem ;
             
     public InputFlagLayout(String name, String comment, boolean optional, String defaultValue, String vipTypeRestriction, String prettyName) {
-        super(name, comment,optional);
-        cbFlagInputItem = new CheckboxItem(prettyName);
+        super(prettyName, comment,optional);
```

Comment:
I'm not sure if that works. Be careful, when the launch button is hit, the *id* must be passed in the input document, not the pretty name. The pretty name may be used only in the GUI. I'm not sure how AbstractSourceLayout uses the name, make sure it is not used for anything else than the GUI.